tests/tools/test_dist_part.py

```diff
@@ -222,6 +166,13 @@ def test_chunk_graph_vector_rows(num_chunks, vector_rows):
     )
 
 
+"""
```

Comment:
remove deprecated code

Reply:
This code should be deprecated using a separate PR#5310.